app/panel/components/Panel.jsx

```diff
@@ -14,18 +14,23 @@
 import React from 'react';
 import ClassNames from 'classnames';
 import Header from '../containers/HeaderContainer';
-import { PlusPromoModal } from '../../shared-components';
+import { PlusPromoModal, InsightsPromoModal, Modal } from '../../shared-components';
```

Comment:
`shared-components` is for components that are used by both the Panel and the Hub. Since InsightsPromoModal is only ever rendered in the Panel, it should live in `panel/components` instead.